tests/ci/docker_images/linux-aarch/build_images.sh

```diff
@@ -2,7 +2,16 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+docker build -t amazonlinux-2-aarch:base amazonlinux-2_base
```

Comment:
How did you come up with this list of images to build and test on? Is it possible we're adding a few too many that are duplicated? Specifically I'm wondering if there is any value in testing multiple versions of the sanitizers with different OS/compilers.

Reply:
FIPS build also enables ASAN. Clang-7 is default compiler in AL2. Clang-9 is used for current sanitizer dimension. Only one CodeBuild dimension is running `run_posix_sanitizers.sh`.

```
    - identifier: ubuntu2004_clang7x_x86_64_fips
      buildspec: ./tests/ci/codebuild/linux-x86/run_fips_tests.yml
      env:
        type: LINUX_CONTAINER
        privileged-mode: true
        compute-type: BUILD_GENERAL1_LARGE
        image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-7x_sanitizer_latest
        variables:
          AWSLC_NO_ASAM_FIPS: 1
```